lib/galaxy_x_gate_app.dart

```diff
@@ -18,16 +21,21 @@ class GalaxyXGateApp extends StatelessWidget {
     return ScreenUtilInit(
       designSize: const Size(375, 812),
       minTextAdapt: true,
-      child: MaterialApp(
-        builder: DevicePreview.appBuilder,
-        title: 'GalaxyXGate',
-        debugShowCheckedModeBanner: false,
-        theme: ThemeData(
-          primaryColor: AppColors.purple,
-          scaffoldBackgroundColor: AppColors.lightBlack,
+      child: MultiBlocProvider(
```

Comment:
why using MultiBlocProvider here ?

Reply:
I prefer working this way because it simplifies the process of accessing different Cubits from different parts of the app if needed.